src/inc/gcinfodecoder.h

```diff
@@ -520,6 +527,9 @@ class GcInfoDecoder
     bool    HasMethodTableGenericsInstContext();
     bool    GetIsVarArg();
     bool    WantsReportOnlyLeaf();
+#if defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
```

Comment:
You should put the `bool    WantsReportOnlyLeaf();` above this also under `#ifdef`